bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -17,16 +17,19 @@
 		<state readOnly="true"/>
 	</channel-type>
 
-	<channel-type id="siren-status">
-		<item-type>String</item-type>
-		<label>Status</label>
+	<channel-type id="siren-status-ro">
```

Comment:
I expect an adjustment in the default translation file (updated key).

Reply:
properties files regenerated